sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventBinding.cs

```diff
@@ -228,11 +230,18 @@ private AuthenticationEventRequestBase GetRequestForEvent(HttpRequestMessage req
         /// <returns>The Event Metadata object.</returns>
         /// <exception cref="AggregateException">Aggregates all the schema validation exceptions.</exception>
         /// <exception cref="Exception">IF the event cannot be determined or if the object model event differs from the requested event on the incoming payload.</exception>
-        private static AuthenticationEventMetadata GetEventAndValidateSchema(string body)
+        internal static AuthenticationEventMetadata GetEventAndValidateSchema(string body)
         {
-            if (!Helpers.IsJson(body))
+            try
+            {
+                if (!Helpers.IsJson(body))
```

Comment:
@KrzysztofCwalina @tg-msft is it ok that we reference Newtonsoft instead of our own JSON parser?

Reply:
No, track 2 libraries should use STJ. Why is this one using Newtonsoft?

Reply:
We are using NS to determine if a string payload is of json type.

Reply:
What is a "track 2" library and why should we stick with STJ?

Reply:
Can you use Utf8JsonReader for it? Just keep reading till false is returned. If there is no exceptions, the playload is JSON. If it's not JSON, and exception will be thrown.

Reply:
track 2 libraries are libraries that follow the following guidelines: https://azure.github.io/azure-sdk/dotnet_introduction.html

Reply:
Thank you, found the guidlines for newtonsoft [here](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-dependencies1). Will remove Newtonsoft

Reply:
Removed newtonsoft dependencies from AuthenticationEvents project

Reply:
@hakimms has removed the usage is using STJ.